src/Sentry/ScopeExtensions.cs

```diff
@@ -307,10 +332,21 @@ public static void AddEventProcessors(this Scope scope, IEnumerable<ISentryEvent
             AttachmentType type = AttachmentType.Default,
             string? contentType = null)
         {
-            var stream = File.OpenRead(filePath);
-            var fileName = Path.GetFileName(filePath);
+            try
+            {
+                var stream = File.OpenRead(filePath);
+                var fileName = Path.GetFileName(filePath);
 
-            scope.AddAttachment(stream, fileName, type, stream.Length, contentType);
+                scope.AddAttachment(stream, fileName, type, contentType);
```

Comment:
If it throws here we don't call dispose on the stream.

Reply:
Good point